symplyphysics/laws/chemistry/electronic_current_in_probe_circuit_in_gas_plasma.py

```diff
@@ -0,0 +1,65 @@
+from sympy import Eq, solve, pi, exp, sqrt
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    symbols,
+    clone_symbol
+)
+from sympy.physics.units import elementary_charge, electron_rest_mass, boltzmann_constant
+
+# Description
+## A gas–discharge plasma is an ionized gas in which the concentrations of positively and negatively charged particles are approximately
+## equal to each other, and the Debye shielding radius is significantly smaller than the characteristic size of the volume in which the
+## ionized gas is located.
+## A probe is an auxiliary metal electrode that is inserted into a plasma volume for its examination. Flat, cylindrical and spherical probes
+## are used. A potential is set relative to one of the electrodes on the probe and the dependence of the incoming current on the value of
+## this potential is removed.
+
+## Law is: I = 0.25 * S * e * n * sqrt(8 * k * T / (pi * m)) * exp(-e * (Uf - U) / (k * T)), where
+## I - probe current,
+## S - probe surface area,
+## e - elementary charge,
+## n - concentration of electrons in plasma,
+## k - boltzmann constant,
+## T - temperature of electrons in plasma,
+## m - electron rest mass,
+## Uf - floating plasma potential,
+## U - potential at the location of the probe.
+
+current = Symbol("current", units.current)
+
+area_probe_surface = Symbol("area_probe_surface", units.area)
+electron_concentration = Symbol("electron_concentration", 1 / units.volume)
+electron_temperature = clone_symbol(symbols.thermodynamics.temperature, "electron_temperature")
+floating_plasma_potential = Symbol("floating_plasma_potential", units.voltage)
+potential_at_location_of_probe = Symbol("potential_at_location_of_probe", units.voltage)
```

Comment:
I think potential is pretty much the same for the entire probe, so you can write `probe_potential`